src/sdk/pynni/nni/compression/torch/quantizers.py

```diff
@@ -203,18 +207,18 @@ def quantize_output(self, output, config, op, **kwargs):
             return output
 
         current_min, current_max = torch.min(output), torch.max(output)
-        op.tracked_min_biased, op.tracked_min = update_ema(op.tracked_min_biased, current_min, op.ema_decay, self.steps)
-        op.tracked_max_biased, op.tracked_max = update_ema(op.tracked_max_biased, current_max, op.ema_decay, self.steps)
-        op.scale, op.zero_point = update_quantization_param(output_bits, op.tracked_min, op.tracked_max)
-        out = self._quantize(output_bits, op, output)
-        out = self._dequantize(op, out)
+        module.tracked_min_biased, module.tracked_min = update_ema(module.tracked_min_biased, current_min, module.ema_decay, self.steps)
+        module.tracked_max_biased, module.tracked_max = update_ema(module.tracked_max_biased, current_max, module.ema_decay, self.steps)
+        module.scale, module.zero_point = update_quantization_param(output_bits, module.tracked_min, module.tracked_max)
+        out = self._quantize(output_bits, module, output)
+        out = self._dequantize(module, out)
         return out
 
     def fold_bn(self, config, **kwargs):
         # TODO simulate folded weight
         pass
 
-    def step(self):
+    def update_step(self):
```

Comment:
Sometimes users reset their step number at epoch begin, sometimes they don't, suggest to add a `step` parameter to allow pruner's step index to be consistent with user training code.  Same for `update_epoch`

Reply:
What should be the value of the step parameter be? I don't think user keeps track of step number ?

Reply:
the step value is an integer.  Normally users do validation or update lr policy per epoch or step, when they do validation/update lr policy per step,  they need to keep track of step number.  the step parameter can be optional.